tests/braille-specs/sw-ke-g2.yaml

```diff
@@ -0,0 +1,5831 @@
+  - ['jumuia', '⠚⠍']
+  - ['kazi', '⠅⠵']
+  - ['kabati', '⠅⠃⠞']
+  - ['kabisa', '⠅⠃⠎']
+  - ['kama', '⠅⠍']
+  - ['kamati', '⠅⠍⠞']
+  - ['kanisa', '⠅⠝⠎']
+  - ['karatasi', '⠅⠗⠞⠎']
+  - ['karibu', '⠅⠗⠃']
+  - ['kaskazini', '⠅⠎⠅⠵']
+  - ['kati', '⠅⠞']
+  - ['katika', '⠅⠞⠅']
+  - ['kidato', '⠅⠙⠞']
+  - ['kijana', '⠅⠚⠝']
+  - ['kila', '⠅⠇']
+  - ['kipakatalishi', '⠅⠏⠅⠞⠩']
+  - ['kitabu', '⠅⠞⠃']
+  - ['kuliko', '⠅⠇⠅']
+  - ['kusini', '⠅⠎⠝']
+  - ['kwa mfano', '⠟⠍⠋']
+  - ['labda', '⠇⠃⠙']
+  - ['lahaja', '⠇⠓⠚']
+  - ['lakini', '⠇⠅⠝']
+  - ['lawama', '⠇⠺⠍']
+  - ['lazima', '⠇⠵⠍']
+  - ['lingana', '⠇⠬⠝']
+  - ['linganisha', '⠇⠬⠝⠩']
+  - ['lugha', '⠇⠣']
+  - ['mbali', '⠭⠇']
+  - ['madhumuni', '⠍⠮⠍⠝']
+  - ['magharibi', '⠍⠣⠃']
+  - ['mahali', '⠍⠓⠇']
+  - ['maisha', '⠍⠩']
+  - ['mara', '⠍⠗']
+  - ['mashariki', '⠍⠩⠅']
+  - ['mbalimbali', '⠭⠇⠭']
+  - ['mfano', '⠍⠋']
+  - ['mjadala', '⠍⠚⠙⠇']
+  - ['moja', '⠍⠚']
+  - ['mpira', '⠍⠏⠗']
+  - ['msichana', '⠍⠎⠡']
+  - ['msikiti', '⠍⠎⠅⠞']
+  - ['mtandao', '⠍⠞⠫']
+  - ['muhimu', '⠍⠓⠍']
+  - ['nafasi', '⠝⠋⠎']
+  - ['namna', '⠝⠍⠝']
+  - ['nini', '⠝⠝']
+  - ['nukta', '⠝⠅⠞']
+  - ['omba', '⠕⠭']
+  - ['okoa', '⠕⠅']
+  - ['ogopa', '⠕⠛⠏']
+  - ['ofisi', '⠕⠋⠎']
+  - ['omboleza', '⠕⠭⠇⠵']
+  - ['pamoja', '⠏⠍⠚']
+  - ['pasipo', '⠏⠎⠏']
+  - ['peleleza', '⠏⠇⠵']
+  - ['pole', '⠏⠇']
+  - ['polepole', '⠏⠇⠏']
+  - ['refu', '⠗⠋']
+  - ['rafiki', '⠗⠋⠅']
+  - ['raha', '⠗⠓']
+  - ['ramani', '⠗⠍⠝']
+  - ['raslimali', '⠗⠎⠇']
+  - ['ratiba', '⠗⠞⠃']
+  - ['runinga', '⠗⠝⠬']
+  - ['rununu', '⠗⠝⠝']
+  - ['sana', '⠎⠝']
+  - ['sababu', '⠎⠃⠃']
+  - ['salamu', '⠎⠇⠍']
+  - ['samahani', '⠎⠍⠓⠝']
+  - ['sambaza', '⠎⠍⠃⠵']
+  - ['sasa', '⠎⠎']
+  - ['sawa', '⠎⠺']
+  - ['sawasawa', '⠎⠺⠎']
+  - ['sehemu', '⠩⠍']
+  - ['serikali', '⠎⠗⠅⠇']
+  - ['shabaha', '⠩⠃⠓']
+  - ['shughuli', '⠩⠣⠇']
+  - ['shukkrani', '⠩⠅⠗⠝']
+  - ['siku', '⠎⠅']
+  - ['simu', '⠎⠍']
+  - ['taratibu', '⠞⠗⠞']
+  - ['tafadhali', '⠞⠋⠮']
+  - ['tarakilishi', '⠞⠗⠅⠩']
+  - ['tarehe', '⠞⠗⠓']
+  - ['tayari', '⠞⠽⠗']
+  - ['teknolojia', '⠞⠅⠝⠚']
+  - ['tena', '⠞⠝']
+  - ['thamani', '⠹⠍⠝']
+  - ['tofauti', '⠞⠋⠞']
+  - ['tovuti', '⠞⠧⠞']
+  - ['ukurasa', '⠥⠅⠗']
+  - ['ujamaa', '⠥⠚⠍']
+  - ['umoja', '⠥⠍⠚']
+  - ['upesi', '⠥⠏⠎']
+  - ['utandawazi', '⠥⠞⠝⠙⠵']
+  - ['uhuru', '⠥⠓⠗']
+  - ['vijana', '⠧⠚⠝']
+  - ['vema', '⠧⠍']
+  - ['vile', '⠧⠇']
+  - ['vilevile', '⠧⠇⠧']
+  - ['virusi', '⠧⠗⠎']
+  - ['vitabu', '⠧⠞⠃']
+  - ['vizuri', '⠧⠵⠗']
+  - ['vuguvugu', '⠧⠛⠧']
+  - ['wasiwasi', '⠺⠎⠺']
+  - ['wakati', '⠺⠅⠞']
+  - ['wakili', '⠺⠅⠇']
+  - ['wala', '⠺⠇']
+  - ['walakini', '⠺⠇⠅⠝']
+  - ['wimbo', '⠺⠭']
+  - ['yaliyomo', '⠽⠇⠍']
+  - ['yaani', '⠽⠝']
+  - ['yamkini', '⠽⠍⠅']
+  - ['yule', '⠽⠇']
+  - ['yuleyule', '⠽⠇⠽']
+  - ['zima', '⠵⠍']
+  - ['zaidi', '⠵⠙']
+  - ['zamani', '⠵⠍⠝']
+  - ['zawadi', '⠵⠺⠙']
+
+###########################################
+# Generated list of words/strings checked as correct by proof reader
+# From the Book of Genesis (Swahili Union Version, Bible Societies of Kenya and Tanzania, 1997)
```

Comment:
Done.
One thing to correct from the impression I gave in the previous version is that the second longer list of generated words (from about line 890) has not been proof read so I have put "unvalidated". I think it is quite useful to have this so that if a change in the table causes a failure in these tests I can investigate further - but I can remove them if you think that is not good practice.

Reply:
OK. I think it's fine to have it if accompanied by the comment.